Lib/fontbakery/commands/check_googlefonts.py

```diff
@@ -4,17 +4,39 @@
 
 from functools import partial
 from fontbakery.specifications.googlefonts import specification
-from fontbakery.commands.check_spec import (
-                                             runner_factory as super_runner_factory
-                                           , main as super_main
-                                           )
-
+from fontbakery.commands.check_spec import main as super_main
+import argparse
+from fontbakery.argumentparser import FBArgumentParser
+import glob
 # The values dict will probably get one or more specific blacklists
 # for the google font project. It would be good if it was not necessary
 # to copy paste this kind of configuration, thus a central init for
 # the google/fonts repository is be good.
 GOOGLEFONTS_SPECIFICS = {}
 
-runner_factory = partial(super_runner_factory, specification, values=GOOGLEFONTS_SPECIFICS)
```

Comment:
Ok, this is needed by the fontbakery- dashboard! I was already wondering. Bring it back!

Reply:
Also add a comment. Same for the `fontbakery.commands.check_spec.runner_factory` removed by #1662 I need this